portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -241,7 +241,11 @@
 	<td>//label[contains(.,'${key_label}')]//following-sibling::div//button[contains(@class,'dropdown-toggle')]</td>
 	<td></td>
 </tr>
-
+<tr>
+	<td>EMPTY_DROPDOWN</td>
```

Comment:
Could you change the name of that path to something like DROPDOWN_ITEM?

Reply:
Done